pymc3/tests/test_distributions.py

```diff
@@ -931,6 +931,7 @@ def test_ex_gaussian(self, value, mu, sigma, nu, logp):
         pt = {'eg': value}
         assert_almost_equal(model.fastlogp(pt), logp, decimal=select_by_precision(float64=6, float32=2), err_msg=str(pt))
 
+    @pytest.mark.xfail(condition=(theano.config.floatX == "float32"), reason="Fails on float32")
```

Comment:
Why was this not come up before? If the problem is scipy having overflow?

Reply:
Because, before both overflow. Now we can use vonmises with larger values of kappa.